src/bms-sender.c

```diff
@@ -0,0 +1,39 @@
+/************************ INCLUDES **************************/
+#include "bms-data.h"
+/************************************************************/
+
+
+BatParams_s BatteryData[NumberOfParameters] = {{"Temperature", TEMPERATURE_MIN , TEMPERATURE_MAX},
+                                                  {"ChargeRate", CHARGERATE_MIN , CHARGERATE_MAX}};
+
+getBmsParamVal_funPtr getParamValue[NumberOfParameters] = {getBmsTemperatue , getBmsChargeRate};
+
+
+
+
+void BmsDataSendToConsole()
+{
+    float BmsData[NumberOfParameters]; 
+    int batteryParamIterater;
+
+    // #ifndef UNIT_TEST
```

Comment:
This is anyway in a compiler directive. Why is it commented as well?
In general, dont leave commented code in commits.